src/proxy/websocketoverhttp.cpp

```diff
@@ -887,19 +880,14 @@ class WebSocketOverHttp::Private : public QObject
 				return;
 		}
 
-		if(reqFrames > 0)
+		if(framesRemoved > 0 || contentRemoved > 0)
 		{
-			q->framesWritten(reqFrames, reqContentSize);
+			q->framesWritten(framesRemoved, contentRemoved);
 			if(self.expired())
 				return;
 		}
 
-		bool hadContent = reqContentSize > 0;
-
-		reqFrames = 0;
-		reqContentSize = 0;
```

Comment:
For consistency, don't reset these here. These and other related vars already get reset the next time they matter, in `update()`. `reqContentSize` was being reset here because `writeBytesAvailable()` was depending on it, but this is no longer the case.